code/modules/clothing/clothing.dm

```diff
@@ -22,7 +23,8 @@
 
 	/// Used by headgear mostly to affect accuracy
 	var/accuracy_mod = 0
-	flags_inventory = NOQUICKEQUIP
+	///Modifies the wearer's max health
+	var/health_mod = 0
```

Comment:
Putting this on clothing now and unused seems odd, easy enough to add it with any relevant gear.

Reply:
oh yeah, forgot to remove this for the pr, will do in a bit.